assets/css/style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 21px 27px;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Montserrat Alternates', sans-serif;
+}
+
+.top-nav {
+  color: #000000;
+  font-size: 20px;
+  text-decoration: none;
+}
+p {
+  font-size: 16px;
+}
+ul,
+li {
```

Comment:
personally, I keep the html content all on the same line (so for me line 23 would look like `ul, li`) but I'm fine doing it this way if you all prefer.

Reply:
I normally do that, too. But it changed automatically like that once I saved